tests/test_web/test_api_v1.py

```diff
@@ -134,6 +134,20 @@ def test_get_builds_invalid_batch(batch, app, client, db):
     assert rv.json == {'error': 'The batch must be a positive integer'}
 
 
+@mock.patch('sqlalchemy.engine.base.Engine.execute')
+def test_get_healthcheck_db_fail(mock_db_execute, app, client, db):
+    mock_db_execute.side_effect = Exception('DB failed')
```

Comment:
Can you change this to `IIBError`?

Reply:
Sure, but why do we expect sqlalchemy to raise IIBError?

Reply:
Sorry, I was just skimming through the diff and misread your mock. Your previous code was correct.

Preferably, please change this to an exception that SQLAlchemy would actually raise.

Reply:
I changed it to a more realistic sqlalchemy error.